src/_imagingft.c

```diff
@@ -1049,6 +1049,7 @@ font_render(FontObject *self, PyObject *args) {
                 int k;
                 unsigned char v;
```

Comment:
```suggestion
```
To fix https://github.com/nulano/Pillow/actions/runs/7007718323/job/19063218352#step:7:237:
```
  src/_imagingft.c:1050:31: warning: unused variable ‘v’ [-Wunused-variable]
   1050 |                 unsigned char v;
        |                               ^
```